CHANGELOG.md

```diff
@@ -67,8 +67,8 @@
 
 ### Changed
 
-- update translations (22.05.2021)
-- Update deltachat-node to `v1.55.0`
+- update translations (02.05.2021)
+- Update deltachat-node to `v1.54.0`
```

Comment:
this change is unrelated and wrong, probably a mistake that got in while rebasing